tuf/tuf_test.go

```diff
@@ -271,68 +248,13 @@ func TestUpdateDelegationsNotEnoughKeys(t *testing.T) {
 	assert.False(t, ok, "no targets file should be created since delegation failed")
 }
 
-func TestUpdateDelegationsReplaceRole(t *testing.T) {
```

Comment:
Since `UpdateDelegations` doesn't take a pointer to a `data.Role`, this scenario is no longer possible